### PR TITLE
feat: add localizedName action to shipping methods

### PR DESCRIPTION
**Pull request by Ibidapo**
#### Summary
This PR adds the action for `setLocalizedName` to the `baseActionsList` for Shipping Methods.

#### Todo

- Tests
  - [x] Unit
  - [ ] Integration
  - [ ] Acceptance
- [ ] Documentation
- [ ] `Type` label for the PR

### CODE REVIEW

**Review by markus-azer — APPROVED** [review]
looks pretty cool 🚀 , thanks @Ibidapo

File: .flowconfig [hunk 1]
```diff
@@ -49,6 +49,11 @@
 .*/node_modules/webpack.*
 .*/node_modules/worker-farm/.*
 .*/node_modules/yargs.*
+.*/node_modules/@babel/.*
+.*/node_modules/normalize-package-data/.*
+.*/node_modules/proxyquire/.*
+.*/node_modules/rollup-plugin-commonjs/.*
+.*/node_modules/rollup-plugin-node-resolve/.*
```
[the remark this answers is not in the document]
Comment (Ibidapo) [reply .flowconfig:56]: I couldn't commit on my local branch, as `flow` was complaining about some unexpected end of input, on the above files. So I was advised me to ignore them.

[the remark this answers is not in the document]
Comment (Ibidapo) [reply .flowconfig:56]: screenshot_2023-03-23_at_13 51 17_720

[the remark this answers is not in the document]
Comment (dogayuksel) [reply .flowconfig:56]: This also works, might be safer way force flow to ignore the `malformed_package_json`, without ignoring the whole package.
```suggestion
.*/node_modules/.*/malformed_package_json/package.json
```

**Review by dogayuksel — APPROVED** [review]

**Pull request merged**